### ISSUE TITLE
[Bug] Cannot read field "f" because "this.tracker" is null

### ISSUE THREAD

**Issue by wling-art** [Insprill/damage-indicator#20]
server version: Purpur git-Purpur-2095 (MC: 1.20.2)
java version: Java 17

```
[23:32:03 ERROR]: Could not pass event EntityRegainHealthEvent to DamageIndicator v1.10.2
java.lang.NullPointerException: Cannot read field "f" because "this.tracker" is null
        at com.zenya.damageindicator.nms.v1_20_R2.ProtocolNMSImpl$HologramImpl.sendPacketToTracked(ProtocolNMSImpl.java:91) ~[[D][伤害指示器]damage-indicator-1.10.2.jar:?]
        at com.zenya.damageindicator.nms.v1_20_R2.ProtocolNMSImpl$HologramImpl.sendCreatePacket(ProtocolNMSImpl.java:64) ~[[D][伤害指示器]damage-indicator-1.10.2.jar:?]
        at com.zenya.damageindicator.nms.v1_20_R2.ProtocolNMSImpl$HologramImpl.spawn(ProtocolNMSImpl.java:55) ~[[D][伤害指示器]damage-indicator-1.10.2.jar:?]
        at com.zenya.damageindicator.event.HologramSpawnEvent.fireEvent(HologramSpawnEvent.java:51) ~[[D][伤害指示器]damage-indicator-1.10.2.jar:?]
        at com.zenya.damageindicator.event.HologramSpawnEvent.<init>(HologramSpawnEvent.java:40) ~[[D][伤害指示器]damage-indicator-1.10.2.jar:?]
        at com.zenya.damageindicator.event.Listeners.onEntityRegainHealthEvent(Listeners.java:82) ~[[D][伤害指示器]damage-indicator-1.10.2.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1860.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[purpur-api-1.20.2-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:77) ~[purpur-api-1.20.2-R0.1-SNAPSHOT.jar:git-Purpur-2095]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[purpur-api-1.20.2-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[purpur-1.20.2.jar:git-Purpur-2095]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[purpur-1.20.2.jar:git-Purpur-2095]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:617) ~[purpur-api-1.20.2-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.world.entity.LivingEntity.heal(LivingEntity.java:1399) ~[?:?]
        at net.minecraft.world.entity.LivingEntity.heal(LivingEntity.java:1388) ~[?:?]
        at net.minecraft.world.entity.animal.horse.AbstractHorse.aiStep(AbstractHorse.java:655) ~[?:?]
        at net.minecraft.world.entity.animal.horse.SkeletonHorse.aiStep(SkeletonHorse.java:152) ~[?:?]
        at net.minecraft.world.entity.LivingEntity.tick(LivingEntity.java:3137) ~[?:?]
        at net.minecraft.world.entity.Mob.tick(Mob.java:461) ~[?:?]
        at net.minecraft.world.entity.animal.horse.AbstractHorse.tick(AbstractHorse.java:690) ~[?:?]
        at net.minecraft.server.level.ServerLevel.tickNonPassenger(ServerLevel.java:1469) ~[?:?]
        at net.minecraft.server.level.ServerLevel.lambda$tick$8(ServerLevel.java:924) ~[?:?]
        at net.minecraft.world.level.entity.EntityTickList.forEach(EntityTickList.java:42) ~[purpur-1.20.2.jar:git-Purpur-2095]
        at net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:901) ~[?:?]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1577) ~[purpur-1.20.2.jar:git-Purpur-2095]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:486) ~[purpur-1.20.2.jar:git-Purpur-2095]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1406) ~[purpur-1.20.2.jar:git-Purpur-2095]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1177) ~[purpur-1.20.2.jar:git-Purpur-2095]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[purpur-1.20.2.jar:git-Purpur-2095]
        at java.lang.Thread.run(Thread.java:840) ~[?:?]
````

**Comment by Insprill** [comment]
Do you have more context for this error? Did it only happen once? Does it happen every time an indicator is shown? Is there a different exception that was thrown earlier?

**Comment by wling-art** [comment]
> Do you have more context for this error? Did it only happen once? Does it happen every time an indicator is shown? Is there a different exception that was thrown earlier?

It's happening all the time, at random intervals, but it's fast, there's no other exception messages about this plugin only this one reporting an error being made.

**Comment by Insprill** [comment]
Alright. I see a rare edge case where this could occur. I'll release an update this weekend.

**Comment by Insprill** [comment]
Fixed in 1.10.3.
